contents/articles/2016-10_michael-barz-ubicomp/index.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Michael Barz's demo at UBICOMP
+date: Mon Oct 17 2016 15:59:02 GMT+0700 (ICT)
```

Comment:
@nathakits - please check dates of both of blog posts to make sure that they have a good order.